pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -467,9 +468,24 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 }
 
 func (d *Driver) supportsVirtiofs() bool {
-	macosVersion, _, err := crcos.RunWithDefaultLocale("sw_vers", "-productVersion")
+	supportsVirtioFS, err := macosNewerThan(11.0)
 	if err != nil {
-		return false
+		log.Debugf("Not able to compare version: %v", err)
 	}
-	return strings.HasPrefix(macosVersion, "12.")
+	return supportsVirtioFS
+}
+
+func macosNewerThan(targetVersion float64) (bool, error) {
+	osVer, err := syscall.Sysctl("kern.osproductversion")
+	if err != nil {
+		return false, err
+	}
```

Comment:
Fwiw, this call could be wrapped in a `var osproductversionOnce = ...`. This would ensure it's only called once for the lifetime of a crc process, and it also makes it easy to override in unit tests for this `macosNewerThan` helper.

Reply:
@cfergeau Done